-1

**Failed Tests**:   Build 
**Summary**: https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-1f2fdc/27501/summary.html
**COMMIT**: fa4c06944514f32364f009522f594b3bb9b6939d
**CMSSW**: CMSSW_12_6_G4VECGEOM_X_2022-09-12-2300/el8_amd64_gcc10
**User test area**: For local testing, you can use `/cvmfs/cms-ci.cern.ch/week0/cms-sw/cmssw/39376/27501/install.sh` to create a dev area with all the needed externals and cmssw changes.


## Build

I found compilation error when building: 

<pre>>> Leaving Package Utilities/RelMon
>> Package Utilities/RelMon built
@@@@ Checking for missing symbols was SKIPPED due to NO_LIB_CHECKING flag in BuildFile: libUtilitiesStaticAnalyzers.so
>> Entering Package src/BigProducts/Simulation
>> Building  big plugin tmp/el8_amd64_gcc10/src/BigProducts/Simulation/pluginSimulation.so
c++: error: objs/el8_amd64_gcc10/SimG4CorePhysicsLists.obj: No such file or directory
gmake: *** [tmp/el8_amd64_gcc10/src/BigProducts/Simulation/pluginSimulation.so] Error 1
>> Leaving Package src/BigProducts/Simulation
>> Package src/BigProducts/Simulation built
>> Leaving Package src/BigProducts/Simulation
>> Package src/BigProducts/Simulation built
</pre>

--- a/SimG4Core/PhysicsLists/src/CMSHadronPhysicsFTFP_BERT.cc
+++ b/SimG4Core/PhysicsLists/src/CMSHadronPhysicsFTFP_BERT.cc
@@ -9,6 +9,7 @@
 
 #include "G4HadronicParameters.hh"
 #include "G4HadronicProcess.hh"
+#include "G4HadronInelasticProcess.hh"
 #include "G4HadProcesses.hh"
 #include "G4SystemOfUnits.hh"
 #include "G4Threading.hh"

